sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcSourceSuite.scala

```diff
@@ -178,6 +178,39 @@ abstract class OrcSuite extends OrcTest with BeforeAndAfterAll {
       checkAnswer(spark.read.orc(path.getCanonicalPath), Row(ts))
     }
   }
+
+  test("SPARK-26859 Append column into middle") {
```

Comment:
I feel that it is not good to just copy the test name from original PR. That test name has its context as it is in AddColumnIntoTheMiddleTest.